README.md

```diff
@@ -4,14 +4,34 @@
 
 ![](https://media.giphy.com/media/FfFvOA9C0h9bhfCuNX/giphy.gif)
 
-#### Notice
-Support storybook v4, v5 and newer
+This addon allows storybook to showcase components with multiple different styled-component themes.
+Supports storybook v4, v5 and newer
 
-#### Installation
+## Installation
 ```bash
 yarn add storybook-addon-styled-component-theme --dev
 ```
 
+## Configuration
+Register the addon in .storybook/main.ts
```

Comment:
if `.ts`, please change the sample to `main.ts`

Reply:
I changed it to main.js as 5.3 throws errors if main is in typescript and i feel like it doesn't matter too much for the example let me know your thoughts though